.github/workflows/springboot-run-stop.yml

```
@@ -0,0 +1,59 @@
+name: Spring Boot Run and Stop
```

actionlint (1.7.4)

"jobs" section is missing in workflow

(syntax-check)